clvm/SExp.py

```diff
@@ -170,7 +164,7 @@ def as_python(self):
         return as_python(self)
 
     def __str__(self):
-        return self.as_bin().hex()
+        return self.atom.hex()
```

Comment:
this won't work for cons objects

Reply:
Yes, seems like this will change test outputs, as `as_bin` did serialization.

Reply:
the intention was to change the test output, or rather, the error messages. But when we actually serialize things with the serializer, we don't use `as_bin()`, we use the serializer. All tests still pass with this change, it's just there is no test for this `__str__` function yet, and it would fail for non-atoms.

Reply:
I'm backing all of this out. I will propose a less intrusive alternative as a separate PR